src/components/show-edit-pod/show-edit-pod.spec.js

```diff
@@ -431,9 +428,8 @@ describe("ShowEditPod", () => {
           onEdit={() => {}}
         />
       );
-      const form = wrapper.find('[data-component="form"]').hostNodes();
-      expect(form.type()).toEqual("form");
-      elementsTagTest(form, ["edit-form"]);
```

Comment:
**question**: Having experimented with these tests a bit I can now see why this test was removed as a result of the new lint rules. But are we sure it's safe to remove? Are we leaving ourself open to regressions? It shouldn't be too hard to rewrite this test block in such a way that the linter is happy but the `elementsTagTest` is retained.

Reply:
@Parsium has removed this, so I assume - yes